### DISCUSSION TITLE
Not getting <laughing> tag in the transcription

### DISCUSSION THREAD

**Discussion by snat1505027** [openai/whisper#1295] [Q&A]
I am trying to get transcriptions for sitcom videos and the important part of the transcription is to include <laugh> tag along with the dialog. I tried the whisper command below to get the transcription, but it does not generate any laugh tag for the part when there is laughter.

`whisper friends_s04e07_seg02_clip_01.mp3 --task transcribe --model medium.en --lang en`

Generated transcription:

> [00:00.000 --> 00:05.800]  I got this sense that she's she's smart and funny and gets you
> [00:07.980 --> 00:09.980]  You got all that from the back of her head
> [00:10.800 --> 00:15.940]  Look, I just think it's time for you to settle down. You know make a choice pick a lane. Who's Elaine?
> [00:21.140 --> 00:26.560]  Little tiny Tarzan swinging on a nose hair swinging with the greatest of ease
> [00:30.520 --> 00:32.520]  Darn it now I'm gonna get to the next verse
> [00:33.200 --> 00:35.200]  Well, you can just go
> [00:35.560 --> 00:39.440]  Greatest of these bah-mah-mah-mah-mah-mah and then go right into it
> [00:42.440 --> 00:45.320]  Wait, how do you know about bah-mah-mah-mah-mah?
> [00:47.120 --> 00:49.340]  You know, I used to play oh
> [00:50.200 --> 00:56.360]  That's right. The keyboards, huh? Yeah, I mean just a little in high school, but then I really got into it in college
> [00:56.360 --> 00:58.940]  I mean, that's that's when I really found my sound
> [01:00.000 --> 01:02.000]  bah-mah-mah-mah-mah

What I want is something like this:

> [00:00.000 --> 00:05.800]  I got this sense that she's she's smart and funny and gets you
> [00:05.800 --> 00:07.980]  (laughing)
> [00:07.980 --> 00:09.980]  You got all that from the back of her head
> [00:09.980 --> 00:10.800]  (laughing)
> [00:10.800 --> 00:15.940]  Look, I just think it's time for you to settle down. You know make a choice pick a lane. Who's Elaine?
> [00:15.940 --> 00:21.140]  (laughing)
> .....

Is it possible to get the laugh tags as well? Thanks in advance.

**Comment by phineas-pta** [comment]
this requires a VAD model with laugh detection, or fine-tuning whisper

**Comment by phineas-pta** [comment]
update: u can try https://github.com/YuanGongND/whisper-at

**Reply by Shivansh-yadav13** [reply]
@phineas-pta `whisper-at` does not recognize laugh

**Reply by phineas-pta** [reply]
raise issue in their repo, im not the devs

**Reply by andupotorac** [reply]
This is useful, thanks for sharing!

**Reply by andupotorac** [reply]
It does, check: https://github.com/YuanGongND/whisper-at/issues/7.

**Comment by Shivansh-yadav13** [comment]
@snat1505027 I'm also looking for a similar solution, did you find it?

**Reply by andupotorac** [reply]
The above works https://github.com/YuanGongND/whisper-at/issues/7
